src/libs/Navigation/NavigationRoot.js

```diff
@@ -30,7 +30,13 @@ class NavigationRoot extends Component {
         }
 
         const path = getPathFromState(state, linkingConfig.config);
-        Log.info('Navigating to route', false, {path});
+
+        // Don't log the route transitions from OldDot because they contain authTokens
+        if (path.includes('/transition/')) {
+            Log.info('Navigating from transition link from OldDot using short lived authToken');
```

Comment:
Actually I don't think we need to log this immediately though so we can add a `false` like we are doing in the other call?

Reply:
Actually wait theres no need, it's false by default: https://github.com/Expensify/expensify-common/blob/master/lib/Logger.jsx#L82